### PR TITLE
Added algorithm bubble_sort in Java

### LINKED ISSUE #601: Bubble Sort Algorithm in Java
#### This issue is about Bubble Sort in Java(https://de.wikipedia.org/wiki/Bubblesort).

- [x] Issue Title is in form `Algo/DS Name [Language]`.
- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This Algo/DS is worth adding to this repository.
- [x] Assign this issue to me (I want to work on this).
	- [x] I ensure that I am not already assigned to 2 or more issues.
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by LoganDungeon**
https://de.wikipedia.org/wiki/Bubblesort

Fixes #601

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [x] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [x] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

**Comment by singhpratyush** [comment]
Hi! Thanks for showing interest in contributing to this repository.

Please make sure that you have ticked the points mentioned in PR description correctly.

Thanks again!

File: bubble_sort/BubbleSort.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+public class BubbleSort {
+
+    private static void sort(int[] list){
```
Comment (singhpratyush) [inline bubble_sort/BubbleSort.java]: `list){` -> `list) {`

File: bubble_sort/BubbleSort.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        if(list == null || list.length == 0 || list.length == 1)
```
Comment (singhpratyush) [inline bubble_sort/BubbleSort.java]: Please change
```java
if(...)
    return;
```
to 
```java
if (...) {
    return;
}
```

File: bubble_sort/BubbleSort.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            return;
+        boolean swapped;
+        int j = 1;
+        do{
```
Comment (singhpratyush) [inline bubble_sort/BubbleSort.java]: `do{` -> `do {`

File: bubble_sort/BubbleSort.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            swapped = false;
+            for (int i = 0; i < list.length - j; i++) {
+                if(list[i] > list[i+1]){
```
Comment (singhpratyush) [inline bubble_sort/BubbleSort.java]: `if(...){` -> `if (...) {`

File: bubble_sort/BubbleSort.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+                    int temp = list[i];
+                    list[i] = list[i+1];
+                    list[i+1] = temp;
+                    swapped = true;
+                }
+            }
+            j++;
+        }
+        while (swapped);
```
Comment (singhpratyush) [inline bubble_sort/BubbleSort.java]:
```java
}
while (swapped);
```
to
```java
} while (swapped);
```

File: bubble_sort/BubbleSort.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    public static void main(String[] args){
+        int[] list = { 8, 4, 6, 1, 10, 7, 2, 5, 9, 3 };
```
Comment (singhpratyush) [inline bubble_sort/BubbleSort.java]: `{ 8, ...3 };` -> `{8, ...3};`

**Comment by aashutoshrathi** [comment]
@LoganDungeon please add the entry to README in the same PR.

File: bubble_sort/BubbleSort.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
+public class BubbleSort {
+
+    private static void sort(int[] list) {
```
Comment (singhpratyush) [inline bubble_sort/BubbleSort.java]: This needs to be public.

**Pull request closed without merging**